src/app/main.cpp

```diff
@@ -16,6 +16,7 @@
  *                                                                         *
  ***************************************************************************/
 
+#include "platformutilities.h"
```

Comment:
Needed?

Reply:
@m-kuhn , the PR includes one commit that removes a few #include from qgismobileapp.h. As a result, that #include was needed in main.cpp. Just a bit of house cleanup really :)